mlir/lib/Dialect/SparseTensor/Transforms/SparseTensorCodegen.cpp

```diff
@@ -925,9 +899,7 @@ class SparseExpandConverter : public OpConversionPattern<ExpandOp> {
     // Determine the size for access expansion (always the innermost stored
     // level size, translated back to original dimension). Note that we
     // recursively rewrite the new DimOp on the **original** tensor.
-    // FIXME: `toOrigDim` is deprecated.
-    const Dimension innerDim = toOrigDim(srcType, srcType.getLvlRank() - 1);
-    const auto sz = sizeFromTensorAtDim(rewriter, loc, desc, innerDim);
+    const auto sz = desc.getLvlSize(rewriter, loc, srcType.getLvlRank() - 1);
```

Comment:
@aartbik Is it the right change? According to the comments, it determine the size according to the innnermost level size, but I am not sure whether it stays true for BSR

Reply:
This looks correct, since this really should be the size of the "innermost" index in the tensor index expression following the iteration graph, so the level is now innermost.

I would update the comment though:

// Determine the size for access expansion (always the innermost stored level size).

Since the translation no longer happens